### PR TITLE
To-do image resize in TF

### PR DESCRIPTION
**Pull request by abhinavsp0730**
Fixed image resize

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

File: chapter_linear-networks/image-classification-dataset.md [hunk 1]
```diff
@@ -277,15 +277,30 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
 
 ```{.python .input}
-#@tab tensorflow
+#tab tensorflow
 def load_data_fashion_mnist(batch_size, resize=None):  #@save
     """Download the Fashion-MNIST dataset and then load into memory."""
     # TODO: Resize
     (mnist_train_x, mnist_train_y), (mnist_test_x, mnist_test_y) = tf.keras.datasets.fashion_mnist.load_data()
-    return (
+    if(resize==None):
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:285]: `resize is None`

File: chapter_linear-networks/image-classification-dataset.md [hunk 2]
```diff
@@ -277,15 +277,30 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
+      return (
         tf.data.Dataset.from_tensor_slices(
             (mnist_train_x, mnist_train_y)).batch(batch_size).shuffle(len(mnist_train_x)),
         tf.data.Dataset.from_tensor_slices((mnist_test_x, mnist_test_y)).batch(batch_size))
+    else:
+      mnist_train = tf.data.Dataset.from_tensor_slices((mnist_train_x, mnist_train_y))
+      mnist_test = tf.data.Dataset.from_tensor_slices((mnist_test_x, mnist_test_y))
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:292]: Move common code to upper?

File: chapter_linear-networks/image-classification-dataset.md [hunk 3]
```diff
@@ -277,15 +277,30 @@ def load_data_fashion_mnist(batch_size, resize=None):  #@save
...
+      def map_fn(img,label):
+        img = tf.reshape(img, (28,28,1))
+        img =  tf.image.resize(img, (resize))
+        return (img,label)
+      mnist_train = mnist_train.map(map_fn)
```
Comment (terrytangyuan) [inline chapter_linear-networks/image-classification-dataset.md:297]: Chain these methods together
